lib/routes/account.js

```diff
@@ -1527,6 +1613,93 @@ module.exports = function (
     },
     {
       method: 'POST',
+      path: '/account/login/send_unblock_code',
+      config: {
+        validate: {
+          payload: {
+            email: validators.email().required()
```

Comment:
this endpoint calls [features.isSigninUnblockEnabledForUser](https://github.com/mozilla/fxa-auth-server/pull/1471/files#diff-ab87c506e682e952973178a41792df12R1652) which depends on a `metricsContext` being available in the payload, can you add that here?